identity/identity.go

```diff
@@ -20,6 +20,13 @@ import (
 	"github.com/ory/kratos/x"
 )
 
+type State uint8
```

Comment:
As discussed on slack, if you do:

```
// swagger:model identityState
type State uint8
```

This should all work! Also, I think it would be a good idea to have the state as a string as that is easier for JSON clients to consume / understand than ints :)